src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -2,7 +2,7 @@
   
   <PropertyGroup Label="Targets required for unit tests to run">
     <TargetFrameworkNetDesktop461>net461</TargetFrameworkNetDesktop461>
-    <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
+    <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
```

Comment:
Why is this made? Is it because of 2.1 obsolete warning?
It's a possible breaking change, since .NET 2.1 projects will now reference the NetStandard2.0 binaries instead. There are some API breaking changes (although they might only affect NetFx). Either way, should be reverted for now.

Reply:
Yes, it was because of the obsolete warning. I did not think there would be side-effects, as the label states that it's only for unit tests?